notebooks/en/function_calling_fine_tuning_llms_on_xlam.ipynb

```diff
@@ -0,0 +1,14340 @@
+{
```

Comment:
<p>Why do we use &lt;small&gt;? I'd remove it from all the possible places since it's difficult to see the text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>